appveyor.yml

```diff
@@ -57,6 +57,8 @@ environment:
   # "(converted from warning) unable to identify current timezone 'C':" for an unknown reason.
   # This environment variable works around to test SparkR against a higher version.
   R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+  # AppVeyor doesn't have python3 yet
+  PYSPARK_PYTHON: python
```

Comment:
I piggy-backed this because I want to make AppVeyor success in this PR.

Reply:
hm, so AppVeyor was broken now?

Reply:
Yes. It has been broken for a while, so the recent `Python3` change is not discovered correctly.